apps/ledger-live-desktop/tests/specs/speculos/receive.address.spec.ts

```diff
@@ -49,3 +49,70 @@ for (const account of accounts) {
     );
   });
 }
+
+test.describe("Receive", () => {
+  const account = Account.TRX_2;
+  test.use({
+    userdata: "speculos-tests-app",
+    speculosCurrency: account.currency,
+  });
+  test(
+    `${account.currency} empty balance Receive displays address activation warning message`,
```

Comment:
I get `[object Object] empty balance Receive displays address activation warning message` here as a title when execute the test

Reply:
updated